### ISSUE TITLE
Add Global Marker Styles options

### ISSUE THREAD

**Issue by mathetos** [WordImpress/google-maps-builder#29]
Custom marker shapes and colors is a great option in the free plugin. But if you have TONS of markers, and want to use that feature you'd have to do that individually which would take forever. Setting a global shape/color for the whole map would be really helpful for users. See here for my suggestion for implementation.

![maps-builder-global-marker-styles](https://cloud.githubusercontent.com/assets/930724/8921365/5806c8e8-3489-11e5-91df-4a00c1211878.png)

**Comment by DevinWalker** [comment]
Good idea. I think the UX needs some though but I like this idea for bulk setting markers.
